src/quoridor/server/server.go

```diff
@@ -48,11 +50,13 @@ func createGameHandler(w http.ResponseWriter, r *http.Request) {
 		response.SendBadRequestError(w, err)
 		return
 	}
-	game, err := gamecontroller.CreateGame(configuration)
+	gameToken:= shortuuid.New()
+	game, err := gamecontroller.CreateGame(configuration, gameToken)
```

Comment:
You should let user to create a game without assign a player to it (without token)